_posts/2020-12-17-solidity-v0.8.0-release-announcement.md

```diff
@@ -0,0 +1,206 @@
+---
+layout: post
+published: true
+title: 'Solidity 0.8.0 Release Announcement'
+date: '2020-12-17'
+author: Solidity Team
+category: Releases
+---
+
+[Solidity 0.8.0](https://github.com/ethereum/solidity/releases/tag/v0.8.0) is a breaking release of the Solidity compiler and language.
+
```

Comment:
Should we refer to the "preview 0.8 release" and mark it superseded, and also perhaps mention if we did a major change to the features contained within?

Reply:
Sounds like a good idea. We could mark new changes in bold or something like that in the changelog at the bottom of the post.

Reply:
I think marking entries bold could look weird.

Reply:
I don't expect many have tried the preview release.

Reply:
I added a sentence wrt the preview release for clarification.